common/api/core-backend.api.md

```diff
@@ -4932,7 +4932,7 @@ export class StandaloneDb extends BriefcaseDb {
     static findByKey(key: string): StandaloneDb;
     // (undocumented)
     get isStandalone(): boolean;
-    static openFile(filePath: LocalFileName, openMode?: OpenMode, options?: SnapshotDbOpenArgs): StandaloneDb;
+    static openFile(filePath: LocalFileName, openMode?: OpenMode, options?: SnapshotDbOpenArgs, upgradeOptions?: UpgradeOptions): StandaloneDb;
```

Comment:
Please consider adding upgradeOptions to SnapshotDbOpenArgs to avoid stuff like `const targetDb = StandaloneDb.openFile(targetDbPath, undefined, undefined, {profile: ProfileOptions.Upgrade});`